srcalc/src/main/webapp/css/srcalc.css

```diff
@@ -94,6 +94,12 @@ footer {
 	display: none;
 }
 
+/* Provide a monospace font for VistA-retrieved notes so that the users are more familiar
+    with the format */
+.patientNote {
```

Comment:
Actually, I think you may have misunderstood me earlier: I'm not sure if Health Factors (or Medications) should be monospaced.